VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimStatus/ClaimTimeline/ClaimPhase.tsx

```diff
@@ -124,39 +108,16 @@ function ClaimPhase({ phase, current, attributes, claimID }: ClaimPhaseProps) {
     )
   }
 
-  const fileRequestsPress = () => {
-    logAnalyticsEvent(Events.vama_claim_review(claimID, attributes.claimType, count))
-    navigateTo('FileRequest', { claimID })
-  }
-
   return (
     <AccordionCollapsible
       noBorder={true}
-      header={getPhaseHeader()}
-      expandedContent={getPhaseExpandedContent()}
-      hideArrow={!phaseLessThanEqualToCurrent}
```

Comment:
We should maybe remove this functionality entirely from AccordionCollapsible as this was the only place we were using it.

Reply:
We still need it for now since it remains in the DEPRECATED_ClaimPhase.tsx component while the feature flag is in place. But we could remove it when we remove the feature flag and the deprecated component